cmd/notation/inspect.go

```diff
@@ -85,6 +95,8 @@ Example - Inspect signatures on an OCI artifact identified by a digest and outpu
 	opts.LoggingFlagOpts.ApplyFlags(command.Flags())
 	opts.SecureFlagOpts.ApplyFlags(command.Flags())
 	cmd.SetPflagOutput(command.Flags(), &opts.outputFormat, cmd.PflagOutputUsage)
+	command.Flags().BoolVar(&opts.allowReferrersAPI, "allow-referrers-api", false, "[Experimental] use the Referrers API to inspect signatures, if not supported, fallback to the Referrers tag schema")
```

Comment:
`command.Flags().BoolVar(&opts.allowReferrersAPI, "allow-referrers-api", false, "[Experimental] use the Referrers API to${operation} signatures, if not supported, fallback to the Referrers tag schema")` is repeated 3 time should we create a helper function for this?